community/kernel/src/test/java/org/neo4j/graphdb/LabelsAcceptanceTest.java

```diff
@@ -86,7 +87,9 @@
 public class LabelsAcceptanceTest
 {
     @Rule
-    public ImpermanentDatabaseRule dbRule = new ImpermanentDatabaseRule();
+    public final ImpermanentDatabaseRule dbRule = new ImpermanentDatabaseRule();
+    @Rule
+    public final TestDirectory testDirectory = TestDirectory.testDirectory();
```

Comment:
that's cause and not even discussable since chain rule should solve exactly that, but it also nice to combine those rules into one that I was referring. 
I do not think that order of them are matter in this test, right?